ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -18,14 +18,16 @@ SCORECARD_CONFIG="${SCORECARD_CONFIG:-/tmp/config/scorecard-basic-config.yml}"
 
 NAMESPACE=$(grep "install_namespace:" "${SHARED_DIR}"/oo_deployment_details.yaml | cut -d ':' -f2 | xargs)
 
-OPERATOR_DIR="${ARTIFACT_DIR}/test-operator-basic"
+pushd "${ARTIFACT_DIR}"
+OPERATOR_DIR="test-operator-basic"
 
 echo "Starting the basic operator-sdk scorecard test for ${OO_BUNDLE}"
 
 echo "Extracting the operator bundle image into the operator directory"
 mkdir -p "${OPERATOR_DIR}"
 pushd "${OPERATOR_DIR}"
 oc image extract "${OO_BUNDLE}" --confirm -a "${OPENSHIFT_AUTH}"
+chmod -R go+r
```

Comment:
I'm seeing a failure in the rehearsal job for this line:
```
ERRO[2021-06-01T22:36:30Z] + chmod -R go+r                              
ERRO[2021-06-01T22:36:30Z] chmod: missing operand after 'go+r'
```
I'm guessing that changing the line to something like this would work:
```suggestion
chmod -R go+r ./
```

Reply:
That would work.  Updated